Windows CI binary build fails due to zlib 1.3.2 library naming change

The `build-executable` workflow fails on Windows because zlib 1.3.2 changed its cmake output file names:

| Old (<=1.3.1) | New (1.3.2) |
|---|---|
| `zlibstatic.lib` | `zs.lib` |
| `zlib.lib` | `z.lib` |
| `zlib.dll` | `z.dll` |

SPC's zlib builder expects the old names, so the static library never gets installed correctly. PHP's configure can't find zlib, and the final link fails with 12 unresolved symbols (inflate, deflate, etc.) from curl, openssl, and libssh2.

```
libcurl_a.lib(content_encoding.obj) : error LNK2019: unresolved external symbol inflate
libcurl_a.lib(content_encoding.obj) : error LNK2019: unresolved external symbol deflateEnd
...
```

Upstream issue filed: https://github.com/crazywhalecc/static-php-cli/issues/1039

In the meantime the workflow needs a workaround to rename the files and re-run configure + nmake.

--- a/.github/pull_request_template.md
+++ b/.github/pull_request_template.md
@@ -1,0 +1,13 @@
+<!--- IMPORTANT - READ BEFORE SUBMITTING YOUR PR -->
+<!--- If you have not discussed this change with us this change before -->
+<!--- submitting a Pull Request, you may be duplicating work already in -->
+<!--- progress. Please either open an issue (for bugs), or create a -->
+<!--- discussion (for features), and discuss the change BEFORE working -->
+<!--- on it, so you are not wasting your time... -->
+
+## PR submitter checklist
+
+- [ ] I have read [CONTRIBUTING.md](../CONTRIBUTING.md)
+- [ ] I discussed this <bug|feature> with the maintainers in #<issue_number> (complete as appropriate)
+- [ ] I have added appropriate tests
+- [ ] I confirm that I have the right to submit this under the project's open source licence

--- a/.github/workflows/build-assets.yml
+++ b/.github/workflows/build-assets.yml
@@ -76,7 +76,7 @@ jobs:
           - ubuntu-24.04-arm
           - macos-15-intel
           - macos-26
-          - windows-2025
+          # - windows-2025 # disabled - https://github.com/crazywhalecc/static-php-cli/issues/1039
     permissions:
       # id-token:write is required for build provenance attestation.
       id-token: write

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -1,0 +1,30 @@
+# Contributing to PIE
+
+Thank you for taking an interest in contributing to PIE.
+
+> [!IMPORTANT]
+> The TL;DR version of this is please either [open an issue](https://github.com/php/pie/issues) if you think you have
+> found a bug, or [create a discussion](https://github.com/php/pie/discussions) if you have an idea for a new feature,
+> **BEFORE** starting work on any code changes. Submitting a pull request without checking first may be duplicating
+> work that is already in progress!
+
+## Reporting bugs
+
+If you think you have a bug, please [open an issue](https://github.com/php/pie/issues), and include:
+
+ - What platform and PIE version you're using
+ - The steps to reproduce the issue - please use the `-v` verbose flag (or higher) 
+ - Complete error messages and/or logs, including stack traces (hint: use `-v` for verbose), but please redact sensitive information
+
+## Submitting PRs
+
+Please **do not** just submit PRs for features or ideas without discussing them first. **Start by
+[creating a discussion](https://github.com/php/pie/discussions) first** if there is not already an open discussion or
+issue. If there is already an open discussion or issue, please comment and wait for feedback before starting any work.
+This is because the work may already be in progress or being investigated already. The PIE project is actively being
+developed, and many features are already in discussion or being developed, so if you do not discuss with us first, you
+may be duplicating work already in progress. 
+
+> [!TIP]
+> We try to stay on top of issues already being worked on with the `maintainer investigating` purple label. If you see
+> this on an issue, it is very likely we are already looking into this.

